packages/babel-plugin-proposal-nullish-coalescing-operator/src/index.ts

```diff
@@ -26,7 +30,7 @@ export default declare((api, { loose = false }) => {
         } else if (scope.path.isPattern()) {
           // Replace `function (a, x = a.b ?? c) {}` to `function (a, x = (() => a.b ?? c)() ){}`
           // so the temporary variable can be injected in correct scope
-          path.replaceWith(template.ast`(() => ${path.node})()`);
+          path.replaceWith(template.ast`(() => ${path.node})()` as t.Statement);
```

Comment:
We can probably do something like this:
```suggestion
          path.replaceWith(template.statement.ast`(() => ${path.node})()`);
```
so that it doesn't have to detect at runtime that it's a single statement (or `template.expression.ast` would also work).